2025/04.md

```diff
@@ -27,7 +27,7 @@ gantt
 For meeting times in your timezone, visit [Temporal docs](https://tc39.es/proposal-temporal/docs/) and run the code below in the devtools console.
 
 ```js
-Temporal.ZonedDateTime.from('2025-04-15T10:00[America/New_York]')
+Temporal.ZonedDateTime.from('2025-04-14T10:00[America/New_York]')
```

Comment:
i'm confused, the first day of the meeting is the 15th, not the 14th - it seems correct as-is? maybe the "1 month before" date needs to be fixed to the 15th though

Reply:
@ljharb The "Dates and times" at the top of this doc lists the 14th. And since this is a remote plenary, and the 14th is a Monday, that seems like it's right to me?

Reply:
ahhh ok, the dates are wrong in the mermaid diagram also. i'll take care of it as part of this PR.